data/sheragi/sheragi ships.txt

```diff
@@ -74,3 +128,90 @@ ship "Emerald Sword"
 	"final explode" "nuke residue fast" 25
 	description "An ancient Sheragi warship that was crewed by mutineers that fled from their civil war and escaped into interstellar space. Found orbiting a disconnected star, this is an impressively large ship, fit for the equally large Sheragi. The ship's main weapon is a fusion-pumped gamma-ray laser called the 'Dragonflame Cannon.' Despite the relatively primitive outfits, it's still a very capable war machine due to its size."
 	description "For its original Sheragi masters, this warship was home to around 100 individuals. For others, however, that number more than doubles, bringing it on par with the largest city ships."
+
+ship "Emerald Sword" "Emerald Sword (Fixed)"
+	sprite ship/emeraldsword
+	thumbnail "thumbnail/emeraldsword"
+	outfits
+		"Dragonflame Cannon"
+		"Particle Waveform Turret" 6
+		"Sheragi Anti-Missile" 2
+		
+		"Large EM Battery"
+		"Large Hybrid Cooling"
+		"Electronic Warfare System" 2
+		"Outfits Expansion" 2
+		
+		"Fusion Drive"
+		"Jump Drive"
+
+	turret "Particle Waveform Turret"
+	turret "Particle Waveform Turret"
+	turret "Particle Waveform Turret"
+	turret "Particle Waveform Turret"
+	turret "Particle Waveform Turret"
+	turret "Particle Waveform Turret"
+	turret "Sheragi Anti-Missile"
+	turret "Sheragi Anti-Missile"
+	gun "Dragonflame Cannon"
+
+ship "Black Diamond"
+	sprite "ship/blackdiamond"
+	thumbnail "thumbnail/blackdiamond"
+	attributes
+		category "Fighter"
+		"cost" 200000
+		"shields" 600
+		"hull" 2400
+		"automaton" 1
+		"mass" 50
+		"drag" .9
+		"heat dissipation" .86
+		"outfit space" 152
+		"weapon capacity" 35
+		"engine capacity" 38
+		weapon
+			"blast radius" 24
+			"shield damage" 240
+			"hull damage" 120
+			"hit force" 360
+	outfits
+		"Heavy Ion Cyclotron"
+		"Shard Fabricator" 2
+		
+		"Electronic Warfare System"
+		"Small EM Battery"
+		"Small Hybrid Cooling"
+		
+		"Fission Drive"
+	
+	gun 0 -39.5 "Heavy Ion Cyclotron"
+	gun 22 -7.5 "Shard Fabricator"
+	gun -22 -7.5 "Shard Fabricator"
+	"engine" 0 38
+		zoom 1.3
+		over
+	"steering engine" 27 22
+		zoom 0.3
+		angle 225
+		over
+		right
+	"steering engine" 27 2
+		zoom 0.3
+		angle 315
+		over
+		left
+	"steering engine" -27 22
+		zoom 0.3
+		angle 135
+		over
+		left
+	"steering engine" -27 2
+		zoom 0.3
+		angle 45
+		over
+		right
+	explode "tiny explosion" 10
+	explode "small explosion" 20
+	"final explode" "final explosion small"
+	description "The Sheragi, being too large to effectively pilot ships this small, invested heavily on the development of autonomous fighters. The hardware that controls this isn't much more complex than the combat drones in human space, but it is much bigger, and the lack of life support systems and amenities for a pilot allows it to carry much more extra outfits."
```

Comment:
```suggestion
	description "The Sheragi, being too large to effectively pilot ships this small, invested heavily in the development of autonomous fighters. The hardware that controls this isn't much more complex than the combat drones in human space, but it is bigger, and the lack of life support systems and amenities for a pilot allows it to carry extra outfitting space."
```
Triple `more`, and two of those were `much more`. If you don't like this rewrite, I'd definitely still recommend removing/substituting some words in the last sentence to get rid of the repetitiveness.